app/services/comment_notifications.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class CommentNotifications
+  attr_reader :comment_id, :model_name
+
+  def initialize(args)
```

Comment:
If the `comment_id` and `model_name` args are always required, I think we should make them named params:
`def initialize(comment_id:, model_name:)`